File: log/logserv.php
<?php

$ctx = new ZMQContext();
$in = $ctx->getSocket(ZMQ::SOCKET_PULL);
$in->bind("tcp://*:5555");

while(true) {
    $messages = $in->recvMulti();
    // ... write the messages to disk or similar in one batch
    echo "Messages Received: ", count($messages), PHP_EOL;
    foreach($messages as $msg) {
        $a = json_decode($msg);
        echo date("Y-m-d h:i:s", $a->time), " ", $a->msg;
        echo PHP_EOL;
    }
}

File: log/logserv.erl
-module(logserv).
-export([run/0]).

run() -> 
  {ok, Ctx} = erlzmq:context(),
  {ok, In} = erlzmq:socket(Ctx, [pull, {active,true}]),
  ok = erlzmq:bind(In, "tcp://*:5555"),
  loop(In).

loop(In) ->
  receive
    {zmq, In, Msg, _Flags} ->
      {ok,{obj, [{_, Time}, {_, Log}] }, [] } = rfc4627:decode(Msg),
      io:format("~B ~s~n", [Time, Log]),
      loop(In)
  end.